app/src/main/AndroidManifest.xml

```diff
@@ -64,10 +65,20 @@
             android:name="com.facebook.sdk.ApplicationId"
             android:value="@string/facebook_app_id" />
 
-        <meta-data android:name="com.facebook.sdk.ApplicationId" android:value="@string/facebook_app_id"/>
-        <provider android:authorities="com.facebook.app.FacebookContentProvider443391136243627"
+        <provider
             android:name="com.facebook.FacebookContentProvider"
+            android:authorities="com.facebook.app.FacebookContentProvider443391136243627"
             android:exported="true" />
-    </application>
 
+        <service android:name=".MyFirebaseMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+            </intent-filter>
+        </service>
+        <!--<service android:name=".FirebaseIDService">-->
```

Comment:
did you mean to leave this commented out?

Reply:
yes, still working on that part!! :)

Reply:
ok, cool :)